src/components/sbb-pearl-chain/sbb-pearl-chain.helper.tsx

```diff
@@ -1,7 +1,5 @@
-import { PtRideLeg } from '../../global/interfaces/pearl-chain-properties';
-
 // eslint-disable-next-line @typescript-eslint/naming-convention
-export const isRideLeg = (leg: any): leg is Extract<PtRideLeg, { __typename: 'PTRideLeg' }> => {
+export const isRideLeg = (leg: any): leg is Extract<any, { __typename: 'PTRideLeg' }> => {
```

Comment:
Can `any` really not be prevented?

Reply:
I removed the hole helper because we need it only one